spec/nexpose/connection/site_spec.rb

```diff
@@ -18,6 +18,7 @@
       let(:basic_site) do
         Nexpose::Site.new('test site name 1').tap do |site|
           site.description = 'test site description 1'
+          site.included_scan_targets[:addresses] = ['localhost']
```

Comment:
You can use `site.included_addresses` to be more concise than `included_scan_targets[:addresses]`